docs/modules/configuration/nav.adoc

```diff
@@ -5,3 +5,4 @@
 * xref:storage.adoc[Storage]
 * xref:audit.adoc[Audit]
 * xref:tracing.adoc[Tracing]
+* xref:schema.adoc[Schema]
```

Comment:
Let's only link this up to the documentation once the feature is complete. We have a couple of more PRs to do to implement the other stores before the feature is ready for use.